.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/main.js
 node_modules
 
+/.vs
```

Comment:
This belongs in your global gitignore file

Reply:
I didn't change anything in this file :)